pagopa/io-app
# fix: [IOBP-706] Payment Method management regression

## Short description
This PR fixes a regression introduced with the implementation of the payment method management during the payment flow.

## List of changes proposed in this pull request
- Removed the paymentMethodManagement from the sequenceS monad since it can be None;
- Replaced the old `GradientScrollView` with the new `IOScrollView`

## How to test
Try to perform a payment, until you reach the confirmation screen you should be able to open the authorization webview.
